test/test_tensorrec.py

```diff
@@ -115,6 +115,14 @@ def test_predict_item_repr(self):
         item_repr = self.unbiased_model.predict_item_representation(self.item_features)
         self.assertEqual(item_repr.shape, (self.item_features.shape[0], 10))
 
+    def test_predict_user_bias(self):
+        user_bias = self.standard_model.predict_user_bias(self.user_features)
+        print(user_bias)  # TODO what are expected values?
+
+    def test_predict_item_bias(self):
+        item_bias = self.standard_model.predict_item_bias(self.item_features)
+        print(item_bias)  # TODO what are expected values?
```

Comment:
Not sure what these expected values should be. When running tests, I get
```
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
```
and
```
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
```

Reply:
The length of those arrays is correct, but they shouldn't be 0 after fitting... something strange is going on.

Reply:
Hmm... ok. I'll play around with it some more and see if I can figure out what's up.